src/OpenRiaServices.Client/Test/Client.Test/Data/OperationTests.cs

```diff
@@ -439,6 +442,91 @@ public void Bug706066_CancelInCallback()
             EnqueueTestComplete();
         }
 
+
+
+        class ExceptionHandleSynchronizationContext : SynchronizationContext
```

Comment:
I think this could be moved up to the region Test Classes currently at row 23-59 for a better structure